src/ui/tools/stringmatching.js

```diff
@@ -0,0 +1,66 @@
+import levenshtein from 'js-levenshtein';
```

Comment:
Should we update the FilterOptions component to use these functions? For a bit of history, the whole levenshtein stuff we do actually originated there from when Alexis did searchable facets. It's kind of funny how it's come full circle. I don't think it should be too hard to update and test, but if it is I think it's totally fine if we say it's not worth the developer time.